src/hooks/useDeviceDetector.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { useState, useEffect } from "react"
+
+const DeviceDetector = () => {
+  return navigator.maxTouchPoints === 0
```

Comment:
Good start. Eventually we will want to add more checks, such as user agent, as some laptop platforms do have touch points.